ActindoWorkFlow/Operation/02_ManageTriggers.md

```diff
@@ -39,18 +39,16 @@ No prerequisites to fulfill.
 4. Enter a descriptive name for the trigger in the *Name* field.   
     The entered name is displayed above the trigger input line.
 
-5. Enter the applicable namespace of the desired model in the *Model* field, for example a PIM product.   
+5. Enter the applicable namespace of the desired entity in the *Model* field, for example the entity PIM product corresponds to the data model Actindo.Modules.Actindo.PIM.Models.PIMProduct. Make sure that the model defined in the trigger is used as the data container for the start place of the workflow, see [Create a workflow](../Operation/01_ManageWorkflows.md#create-a-workflow). By entering the model, replace the dots with backslashes, for example **Actindo\Modules\Actindo\PIM\Models\PIMProduct**.   
     The entered namespace is displayed to the right of the trigger name above the trigger input line.
 
-    > [Info] The data models can be taken from the relevant API documentation.
-
-[comment]: <> (Update as soon as menu entry for models exists in Dev Tools)  
+    > [Info] You can take the data models from the related data models under *Dev Tools > API > Tab DATA MODELS*.
```

Comment:
Kurz prüfen, ob es so gemeint ist: You can take the **data models** from the related **data models**...
You can take the name?